website/members/forms.py

```diff
@@ -38,28 +38,7 @@ class Meta:
         ]
         model = Profile
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
```

Comment:
Woah, this init must not go, the widget is set to hidden here and we do not want to lose that functionality.

Reply:
That's a big 'whoops'